projects/elife.yaml

```diff
@@ -2047,12 +2050,40 @@ fastly-logs:
 
 data-pipeline:
     description: Data consolidation project
-    domain: false
-    intdomain: false
+    domain: pipeline
+    intdomain: pipeline
+    formula-repo: https://github.com/elifesciences/data-pipeline-formula
     aws:
-        ec2: false
+        ec2:
+            ami: ami-1d4e7a66 # 16.04
+        ports:
+            - 22
+            - 443
```

Comment:
HTTPS-only... ok

Reply:
just an oversight. I've stuck nifi behind an nginx instance like most everything we use. NiFi is very strict about security, unsurprising given it's origins, and it took a bit of wrangling to ensure the headers were being proxied correctly. I think my browser did the upgrade to https without ever attempting http so it never came across my radar.

Reply:
It is good for me as this `ports` settings is only used on AWS and `anything.elifesciences.org` is pinned to HTTPS anyway. Things port 80 is useful for:

- within Vagrant
- `anything.elife.internal` hostname use
- redirecting to HTTPS